.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 bower_components/
+.idea
```

Comment:
why? i mean IMO we should remove it completely from git repo but ignoring it only won't solve it